charts/edge/templates/statefulset.yaml

```diff
@@ -71,6 +84,12 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.labels['apps.kubernetes.io/pod-index']
+          {{- if .Values.enableK8sLogs }}
+          - name: EDGE_SELF_NODE_NAME
```

Comment:
The var refers to the node that the running instance is on, which is why I was thinking `EDGE_` since that's the name of the executing thing. I can update for consistency though if you think that's better

Reply:
I don't feel too strongly about it. Just noticed it was the first var where we departed on the brand name. 🤷